Tools/TestWebKitAPI/Tests/IPC/IPCSerialization.mm

```diff
@@ -412,6 +415,35 @@ String certDataBase64(""
     return components;
 }
 
+#if HAVE(SEC_KEYCHAIN)
+static SecKeychainRef getTempKeychain()
+{
+    SecKeychainRef keychainRef = NULL;
+    uuid_t uu;
+    char pass[PATH_MAX];
+
+    uuid_generate_random(uu);
+    uuid_unparse_upper(uu, pass);
```

Comment:
pass's length of PATH_MAX is a little strange.  Why not just use NSUUID.UUID and %@?

Reply:
Because I also use `pass` a few lines later as the C-string password for the keychain. (This is code from a previous life which I basically copied/pasted.) I could use a value smaller than `PATH_MAX` to be sure, but that's been the historical value to use when doing anything path-related.